pom.xml

```diff
@@ -154,6 +177,12 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>com.openshift</groupId>
```

Comment:
There's another PR #1 that adds fabric8's client, I would preferred to use that one.

Reply:
ok, will look into that